owasm/example/src/logic.rs

```diff
@@ -43,20 +46,35 @@ mod tests {
     #[test]
     fn test_execute() {
         // Average is 120.00
-        let data1 = Data { coin_gecko: 100.0, crypto_compare: 150.0, binance: 110.0 };
+        let data1 =
+            Data { coin_gecko: 100.0, crypto_compare: 150.0, binance: 110.0, alphavantage: 120.0 };
         // Average is 220.00
-        let data2 = Data { coin_gecko: 200.0, crypto_compare: 250.0, binance: 210.0 };
+        let data2 =
+            Data { coin_gecko: 200.0, crypto_compare: 250.0, binance: 210.0, alphavantage: 220.0 };
         // Average among the two data points is 170.00
         assert_eq!(execute(vec![data1, data2]), Result { price_in_usd: 17000 });
     }
 
+    #[test]
+    fn test_call_stock() {
+        let data = Data::build_from_local_env(&Parameter {
+            symbol: coins::Coins::ETH,
+            alphavantage_symbol: String::from("GOOG"),
+            alphavantage_api_key: String::from("WVKPOO76169EX950"),
+        })
+        .unwrap();
+        println!("Current GOOG price (times 100) is {:?}", execute(vec![data]));
```

Comment:
This line want to test GOOG current price? but your execute function will average eth price and goog price. That is the thing you want to test?